src/main/resources/attackvectors/UnrestrictedFileUploadPayload.properties

```diff
@@ -0,0 +1,38 @@
+<b>How to exploit this level?</b><br/>\
+  Filename validation was improved (add case sensitive security), so now to exploit this level you have to create a file with extension .php, and paste the following code: <br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(navigator.userAgent);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  Now if we upload this file and access it, the file should get executed in the browser. You need to use browser which handle .php files like Use it on Internet Explorer (version OS Build 19042.985).
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_6=Hackers can execute code in a user's machine by uploading a file in the proper format to the server and trick the user into accessing it. Generally, users run personal machines with administrator privileges and disabled browser security policies which make attack execution easier. A persistent XSS attack is a similar type of attack that causes cookie and sensitive data thefts.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation was improved (only files containing .jpeg and .png extension are accepted), so now to exploit this level you have to a create a file with extension .jpeg.html,and paste the following code: <br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(navigator.userAgent);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  Now if we upload this file and access it, the file should get executed in the browser.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_7=Hackers can execute code in a user's machine by uploading a file in the proper format to the server and trick the user into accessing it. Generally, users run personal machines with administrator privileges and disabled browser security policies which make attack execution easier. A persistent XSS attack is a similar type of attack that causes cookie and sensitive data thefts.<br/>\
+<b>How to exploit this level?</b><br/>\
+  Filename validation was improved (only files with .jpeg and .png extension are accepted), so now to exploit this level you have to create a file with extension like .html.jpeg, and paste the following code: <br/>\
+  <code>&lt;html&gt;&lt;script&gt;alert(navigator.userAgent);&lt;/script&gt;&lt;/html&gt;</code><br/>\
+  Now you have to insert a null byte character before .jpeg or .png, then this extension will be ignored and a file will be treated as .html. Now if we upload this file and access it, the file should get executed in the browser.
+UNRESTRICTED_FILE_UPLOAD_PAYLOAD_LEVEL_8=Hackers can execute code in a user's machine by uploading a file in the proper format to the server and trick the user into accessing it. Generally, users run personal machines with administrator privileges and disabled browser security policies which make attack execution easier. A persistent XSS attack is a similar type of attack that causes cookie and sensitive data thefts.<br/>\
```

Comment:
The description needs to be changed as this attack will execute in client's machine.